doc/design.md

```diff
@@ -51,10 +51,58 @@ backend --> customerCluster: manages
 @enduml
 ```
 
+#### EM Provides a REST Interface
+`dsn~rest-interface~1`
+
+EM provides a REST interface to clients.
+
+Covers:
+* [`req~rest-interface~1`](system_requirements.md#em-provides-a-rest-interface)
```

Comment:
Should we think of a tool to help creating these links?
I created an emacs script, but probably only I find it useful 😃

Reply:
I started writing an IntelliJ plugin a while ago. Ran out of time though...

Reply:
It does not need to be a hyperlink to be traced successfully. The hyperlink is a neat add-on, but not strictly required.